go/chr.go

```diff
@@ -0,0 +1,15 @@
+package pehape
+
+import "bytes"
+
+// Chr is a function that returns a character from a number.
+// Parameter :
+// - codepoint => it can be in decimal, octal, or hex values.
+// Return :
+// - single-byte string
+func Chr(codepoint int) string {
```

Comment:
function parameter nggak sesuai dokumentasi

Reply:
Oke pak bentar saya fix

Reply:
Dokumentasi yg mana pak yg miliknya official php kah?

Reply:
ini: "it can be in decimal, octal, or hex values", tapi `codepoint` cuma terima integer, ngga ada contoh atau etst buat octal dan hex

Reply:
oke paham pak